### PR TITLE
Repository page

### PR DESCRIPTION
**Pull request by PaulFarid**
Created The Repo-Page

### CODE REVIEW

**Review by walidoow — CHANGES_REQUESTED** [review]
You should put the add button to the right, and remove the black border of that same button.

image

File: src/app/components/header/header.component.ts [hunk 1]
```diff
@@ -34,7 +34,8 @@ export class HeaderComponent {
   mapPathToTitle(path: string): string {
     const titleMap: { [key: string]: string } = {
       points: 'My Points',
+      repository: 'Repository',
     };
-    return titleMap[path] || 'My Workouts';
+    return titleMap[path] || 'My Calendar';
```
Comment (walidoow) [inline src/app/components/header/header.component.ts]: Why did you change it to my calendar?

File: src/app/app.module.ts [hunk 2]
```diff
@@ -19,7 +18,9 @@ import { PointsComponent } from './pages/points/points.component';
   imports: [
     BrowserModule,
     AppRoutingModule,
-    MatIconModule
+    MatIconModule,
+   
+
```
Comment (walidoow) [inline src/app/app.module.ts]: remove the extra space

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -12,6 +12,9 @@ const routes: Routes = [
     path: '',
     component: LandingComponent,
   },
+  {path: 'repository',
```
Comment (walidoow) [inline src/app/app-routing.module.ts]: return the line, keep everything consistent

File: angular.json [hunk 4]
```diff
@@ -100,5 +100,12 @@
         }
       }
     }
+  },
+  "cli": {
+
+    "analytics": "1a777da3-49d1-4907-8312-c6b0c8a12420"
+
+    
+
```
Comment (walidoow) [inline angular.json]: remove this

File: .gitignore [hunk 5]
```diff
@@ -29,3 +29,5 @@ e2e/*.map
 
 # System Files
 .DS_Store/
+
+
```
Comment (walidoow) [inline .gitignore]: remove the extra lines

**Review by walidoow — CHANGES_REQUESTED** [review]

File: src/app/components/header/header.component.ts (hunk 1 above)
Comment (PaulFarid) [reply src/app/components/header/header.component.ts]: probably something happened when  i merged the main

File: src/app/components/header/header.component.ts [hunk 6]
```diff
@@ -36,6 +36,6 @@ export class HeaderComponent {
       points: 'My Points',
       repository: 'Repository',
     };
-    return titleMap[path] || 'My Calendar';
```
Comment (walidoow) [inline src/app/components/header/header.component.ts:41]: @PaulFarid finally, @amczuboka mentioned she was the one who changed it.

**Review by amczuboka — APPROVED** [review]
Once @walidoow requests have been done, it is ready to merge

**Review by IsabelleCzuboka — APPROVED** [review]
Looks good
